### PR TITLE
Fix profile rank graph being hidden (and improve visually)

### PR DESCRIPTION
**Pull request by EVAST9919**
- Uses `User` bindable instead of drawing itself in the ctor
- Supports null user
- Can handle the edge case when we can get zero values in rank history
- Hides graph if we have no rank history but have the rank itself

### CODE REVIEW

File: osu.Game/Overlays/Profile/RankChart.cs [hunk 1]
```diff
@@ -60,52 +70,74 @@ public RankChart(User user)
...
+        private void load(OsuColour colours)
         {
-            rankText.Text = user.Statistics.Rank > 0 ? $"#{user.Statistics.Rank:#,0}" : "no rank";
-            performanceText.Text = user.Statistics.PP != null ? $"{user.Statistics.PP:#,0}pp" : string.Empty;
-            relativeText.Text = $"{user.Country?.FullName} #{user.CountryRank:#,0}";
+            graph.Colour = colours.Yellow;
         }
 
-        private void showHistoryRankTexts(int dayIndex)
+        public void Redraw(User user)
```
Comment (peppy) [inline osu.Game/Overlays/Profile/RankChart.cs]: can this user be a bindable rather than randomly named method?

**Review by smoogipoo — DISMISSED** [review]
This is pretty awkward: https://streamable.com/af19p

Should just display "No recent plays" if the user isn't ranked.

**Comment by EVAST9919** [comment]
i'm in the army atm, can't contribute for 1.5 years :(

**Comment by peppy** [comment]
You should have given us more warning that you were leaving! Never fear; we'll pick up the slack for open PRs.

**Review by peppy — APPROVED** [review]

**Pull request merged**